polaris-icons/.babelrc.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @type {import('@babel/core').TransformOptions}
+ */
 module.exports = {
-  extends: '../babel.config.js',
+  presets: [['@shopify/babel-preset', {react: true}]],
```

Comment:
I'd say leave this as it is. Enabling the typescript transforms in this package has no effect - as there are no ts/tsx files. I don't think this is worth the divergence from the base config.